guardian/bonobo
# Reworked Kong access

## What does this change?

- Removes the need for a secondary LB in order to access Kong admin ports. This is done by resolving their IP addresses directly from EC2
- Adds an authentication token to the Kong admin requests.  This is to support https://github.com/guardian/content-api/tree/ag/kong-infra-improvements; this is looked up each time in order to make rotation simpler.

## How to test

Deploy and test. Make sure that you update the CODE configuration files in-line with the changes in this PR

## How can we measure success?

- Lower running cost due to less LBs
- Simpler to administer because we don't have to copy-paste the internal LB name across; nodes are looked up automatically

## Have we considered potential risks?
